convert_zdf.py

```diff
@@ -0,0 +1,168 @@
+"""Convert ZDF point cloud to the available formats.
+
+Available formats:
+    .ply - Polygon File Format
+    .csv,.txt - [X, Y, Z, r, g, b, Contrast]
+    .png,.jpg,.bmp,.tiff - 2D RGB image
```

Comment:
This is going to be out of date as soon as someone is adding a new format.

Reply:
Sure, but what do you recommend, not list anything? Change to ".png, .jpg, etc"
Alternatively I could change the option to be a list of 2D output formats, and then simply pass them "as is" to the _convert_to_2d function.

Reply:
Perhaps not specify it in the module documentation? For me personally I think it should already be pretty clear from the arguments in the argparse section.

Reply:
I think we should have available formats written at the very top in the description area. Some customers don't care about the code and will not bother reading it. They just want to convert files and to do that they need just to know what formats are available and how to use the script.

Reply:
I see.
In that case leave it be.